client/src/components/App.jsx

```diff
@@ -1,17 +1,26 @@
+/* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import Header from './organisms/Header/Header';
 import HomePage from './pages/HomePage/HomePage';
 import TimeLinePage from './pages/TimeLinePage/TimeLinePage';
-import { BrowserRouter as Router, Route } from 'react-router-dom';
+import PrivateRoute from './molecules/PrivateRoute';
```

Comment:
would be a good time to just rename this file. Super confusing.
Take it one at a time.

Reply:
Change the name `PrivateRoute` or the `molecule` thing?